src/components/component-playground.jsx

```diff
@@ -198,11 +206,17 @@ module.exports = React.createClass({
   },
 
   _renderPreview: function() {
-    return <div ref="previewContainer"
-                key="previewContainer"
-                className={this._getPreviewClasses()}>
-      {this.loadChild('preview')}
-    </div>
+    store.dispatch(changeFixture(this.state.fixtureContents.reduxStore));
```

Comment:
I don't have a solution for this, but it's not a good approach. Maybe `componentWillReceiveProps`?